index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+    
+  
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Technical</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  
+  <main id ="main-doc">
+    <nav id = "navbar" class="menu">
+      <header>Navigation</header>
+    <ul>
+      
```

Comment:
Be careful with blank lines and indentation.

- [ ] Do not add blank lines, spaces, or indentations without a reason.